erpnext/controllers/selling_controller.py

```diff
@@ -191,6 +191,7 @@ def get_item_list(self):
 				frappe.throw(_("Row {0}: Qty is mandatory").format(d.idx))
 
 			if self.has_product_bundle(d.item_code):
+				qty = frappe.db.sql('Select sum(pbi.qty) as qty from `tabProduct Bundle Item` pbi where pbi.parent = %(pb)s ', {'pb': d.item_code}, as_dict=1)[0].qty
```

Comment:
You can use `db.get_value` instead.

`qty = frappe.db.get_value("Product Bundle Item", filters={"parent": d.item_code}, fieldname="sum(qty)")`